initiatives/info.yml

```diff
@@ -0,0 +1,57 @@
+---
+resource: budgets
```

Comment:
rename to `initiatives`

Reply:
fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)